ote_sdk/ote_sdk/usecases/exportable_code/demo/README.md

```diff
@@ -5,54 +5,90 @@ Name of package is a name of model which was deployed.
+1. Install Python (version 3.8 or higher).
 
 2. Install the package in the clean environment:
 ```
-python -m pip install <model_name>-0.0-py3-none-any.whl
+python -m pip install demo_package-0.0-py3-none-any.whl
 ```
 
 
 When the package is installed, you can import it as follows:
 ```
-python -c "from <model_name> import create_model"
+python -c "from demo_package import create_model"
 ```
 
 > **NOTE**: On Linux and macOS, you may need to type `python3` instead of `python`. You may also need to [install pip](https://pip.pypa.io/en/stable/installation/).
 > For example, on Ubuntu execute the following command to get pip installed: `sudo apt install python3-pip`.
 
 ## Usecases
 
-1. You can run demo, using entry-point <model_name>. For example if you model name `atss`, running the application with the -h option yields the following usage message:
+1. Running the `demo.py` application with the `-h` option yields the following usage message:
    ```
-   usage: atss [-h] -i INPUT -m MODEL
+   usage: demo.py [-h] -i INPUT -m MODEL [-c CONFIG]
 
    Options:
      -h, --help            Show this help message and exit.
      -i INPUT, --input INPUT
                            Required. An input to process. The input must be a
-                           single image.
+                           single image, a folder of images, video file or camera
+                           id.
      -m MODEL, --model MODEL
                            Required. Path to an .xml file with a trained model.
+     -c CONFIG, --config CONFIG
+                           Optional. Path to an .json file with parameters for
+                           model.
+
    ```
 
-   As a model, you can use `model.xml` from generated zip.
+   As a model, you can use `model.xml` from generated zip. So can use the following command to do inference with a pre-trained model:
+   ```
+   python3 demo.py \
+     -i <path_to_video>/inputVideo.mp4 \
+     -m <path_to_model>/model.xml
+   ```
 
-2. You can create your own demo application, using <model_name> package.
\ No newline at end of file
+   Also you can define own json config that specify some model parameters. To create this config please see `config.json` in demo_package wheel.
+
+2. You can create your own demo application, using `demo_package`. The main function of package is `create_model`:
+   ```python
+   def create_model(model_path, config_file=None):
+    """
+    Create model using ModelAPI factory
+
+    :param model_path: Path to .xml model
+    :param config_file: Path to .json config. If not define, use config from demo_package
+    """
+   ```
+   Function return model wrapper from ModelAPI. To get more information please see [ModelAPI](https://github.com/openvinotoolkit/open_model_zoo/tree/master/demos/common/python/openvino/model_zoo/model_api).
+
+   Some example how to use `demo_package`:
+   ```python
+   import cv2
+   from demo_package import create_model
+   path_to_model = ""
+   path_to_image = ""
```

Comment:
It looks misleading. I'd suggest to either put some meaningful values here, or simply omit these variables at all and let them undefined in this sample chunk of code.